app/frontend/public/store/ordering/index.html

```diff
@@ -0,0 +1,80 @@
+<!doctype html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <title>eCommerce Ordering</title>
+        <link
+            href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.3/dist/css/bootstrap.min.css"
+            rel="stylesheet"
+            integrity="sha384-QWTKZyjpPEjISv5WaRU9OFeRpok6YctnYmDr5pNlyT2bRjXh0JMhjY6hW+ALEwIH"
+            crossorigin="anonymous"
+        />
+        <link rel="stylesheet" href="style.css" />
+    </head>
+
+    <body>
+        <div class="header">
+            <h1 class="header-title">ATS Store Ordering Page</h1>
+            <div class="header-buttons">
+                <a
+                    href="/store/orders"
+                    class="header-button btn btn-outline-dark"
+                    >Orders</a
+                >
+                <a href="/store" class="header-button btn btn-outline-dark"
+                    >Home</a
+                >
```

Comment:
I think you should increase the contrast of the button, try using btn-light instead of btn-outline-dark

Reply:
Okay fixed!